packages/protocol/contracts/governance/Governance.sol

```diff
@@ -848,21 +869,29 @@ contract Governance is
   }
 
   /**
-   * @notice Checks if a byzantine quorum of validators has whitelisted the given hotfix.
+   * @notice Returns number of validators from current set which have whitelisted the given hotfix.
    * @param hash The abi encoded keccak256 hash of the hotfix transaction.
-   * @return Whether validator whitelist tally >= validator byztanine quorum (2f+1)
+   * @return Whitelist tally
    */
-  function isHotfixPassing(bytes32 hash) public view returns (bool) {
+  function hotfixWhitelistValidatorTally(bytes32 hash) public view returns (uint256) {
     uint256 tally = 0;
     uint256 n = numberValidatorsInCurrentSet();
     for (uint256 idx = 0; idx < n; idx++) {
       address validator = validatorAddressFromCurrentSet(idx);
-      if (hotfixes[hash].whitelisted[validator]) {
+      if (isHotfixWhitelistedBy(hash, validator)) {
```

Comment:
This might be off topic and I don't fully remember, but I believe the validator set contains the addresses of the validating keys, but is that the keys we expect validators to whitelist with? cc @asaj

Reply:
will fix in a followup PR

Reply:
Opened https://github.com/celo-org/celo-monorepo/issues/1832